config/logging.php

```diff
@@ -42,74 +42,66 @@
     |                    "custom", "stack"
     |
     */
-
+    
     'channels' => [
         'stack' => [
-            'channels'          => ['single'],
-            'driver'            => 'stack',
+            'channels' => [
+                'single',
+            ],
+            'driver' => 'stack',
             'ignore_exceptions' => false,
         ],
-
         'single' => [
             'driver' => 'single',
-            'level'  => env('LOG_LEVEL', 'debug'),
-            'path'   => storage_path('logs/system.log'),
+            'level' => env('LOG_LEVEL', 'debug'),
+            'path' => storage_path('logs/system.log'),
         ],
-
         'daily' => [
-            'days'   => 14,
+            'days' => 14,
             'driver' => 'daily',
-            'level'  => env('LOG_LEVEL', 'debug'),
-            'path'   => storage_path('logs/system.log'),
+            'level' => env('LOG_LEVEL', 'debug'),
+            'path' => storage_path('logs/system.log'),
         ],
-
         'slack' => [
-            'driver'   => 'slack',
-            'emoji'    => ':boom:',
-            'level'    => env('LOG_LEVEL', 'critical'),
-            'url'      => env('LOG_SLACK_WEBHOOK_URL'),
+            'driver' => 'slack',
+            'emoji' => ':boom:',
+            'level' => env('LOG_LEVEL', 'critical'),
+            'url' => env('LOG_SLACK_WEBHOOK_URL'),
             'username' => 'Winter Log',
         ],
-
         'papertrail' => [
-            'driver'       => 'monolog',
-            'handler'      => env('LOG_PAPERTRAIL_HANDLER', \Monolog\Handler\SyslogUdpHandler::class),
+            'driver' => 'monolog',
+            'handler' => env('LOG_PAPERTRAIL_HANDLER', \Monolog\Handler\SyslogUdpHandler::class),
```

Comment:
@jaxwilko Laravel actually imports these classes at the top of their files (see https://github.com/laravel/laravel/blob/9.x/config/app.php#L3), which might be a setup that we could encounter (thus failing the file must start with return requirement), is it easy to support that or should we just say that we don't support it?

Reply:
Should be fairly easy to check for and ignore imports prior to a return stmt

Reply:
Support added in https://github.com/wintercms/storm/pull/40/commits/ec040435c486b0a5cb369137cf3d5dfa2d932c2e